service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json

```diff
@@ -0,0 +1,761 @@
+    },
+    "storageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS"
+      ],
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Replication option for the VM image storage account"
+      }
+    },
+    "supportLogStorageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS"
+      ],
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Replication option for the support log storage account"
+      }
+    },
+    "applicationDiagnosticsStorageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS"
+      ],
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Replication option for the application diagnostics storage account"
+      }
+    }
+  },
+  "variables": {
+    "computeLocation": "[parameters('clusterLocation')]",
+    "dnsName": "[parameters('clusterName')]",
+    "vmStorageAccountName": "[concat(parameters('clusterName'),'1')]",
+    "vmName": "vm",
+    "applicationStartPort": "20000",
+    "applicationEndPort": "30000",
+    "ephemeralStartPort": "49152",
+    "ephemeralEndPort": "65534",
+    "publicIPAddressName": "PublicIP-VM",
+    "publicIPAddressType": "Dynamic",
+    "vmStorageAccountContainerName": "vhds",
+    "virtualNetworkName": "VNet",
+    "addressPrefix": "10.0.0.0/16",
+    "subnet1Name": "Subnet-1",
+    "subnet2Name": "Subnet-2",
+    "subnet1Prefix": "10.0.0.0/24",
+    "subnet2Prefix": "10.0.1.0/24",
+    "nicName": "NIC",
+    "lbName": "LoadBalancer",
+    "lbIPName": "PublicIP-LB-FE",
+    "availSetName": "AvailabilitySet",
+    "maxPercentUpgradeDomainDeltaUnhealthyNodes": "100",
+    "vmImagePublisher": "MicrosoftWindowsServer",
+    "vmImageOffer": "WindowsServer",
+    "vmImageSku": "2012-R2-Datacenter",
+    "vmImageVersion": "latest",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnet1Ref": "[concat(variables('vnetID'),'/subnets/',variables('subnet1Name'))]",
+    "supportLogStorageAccountName": "[concat(parameters('clusterName'),'2')]",
+    "applicationDiagnosticsStorageAccountName": "[concat(parameters('clusterName'),'3')]",
+    "lbID0": "[resourceId('Microsoft.Network/loadBalancers',concat(variables('lbName'),'-','0'))]",
+    "lbIPConfig0": "[concat(variables('lbID0'),'/frontendIPConfigurations/LoadBalancerIPConfig')]",
+    "lbPoolID0": "[concat(variables('lbID0'),'/backendAddressPools/LoadBalancerBEAddressPool')]",
+    "lbProbeID0": "[concat(variables('lbID0'),'/probes/FabricGatewayProbe')]",
+    "lbHttpProbeID0": "[concat(variables('lbID0'),'/probes/FabricHttpGatewayProbe')]",
+    "vmNodeType0Name": "[concat('NT1', variables('vmName'))]",
+    "vmNodeType0Size": "Standard_A2"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-05-01-preview",
```

Comment:
Wont fix – Any changes to the schema will need to be validated by the product team, so if we do end up moving to a GA version. I will submit a change then. I do not want to make it a variable for now.

From: Kay Singh [mailto:notifications@github.com]
Sent: Tuesday, December 1, 2015 2:45 PM
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Chacko Daniel <chackdan@microsoft.com>
Subject: Re: [azure-quickstart-templates] WAD 5 Node (#863)


In service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/863#discussion_r46351555>:

> +    "vmImageVersion": "latest",

> +    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",

> +    "subnet1Ref": "[concat(variables('vnetID'),'/subnets/',variables('subnet1Name'))]",

> +    "supportLogStorageAccountName": "[concat(parameters('clusterName'),'2')]",

> +    "applicationDiagnosticsStorageAccountName": "[concat(parameters('clusterName'),'3')]",

> +    "lbID0": "[resourceId('Microsoft.Network/loadBalancers',concat(variables('lbName'),'-','0'))]",

> +    "lbIPConfig0": "[concat(variables('lbID0'),'/frontendIPConfigurations/LoadBalancerIPConfig')]",

> +    "lbPoolID0": "[concat(variables('lbID0'),'/backendAddressPools/LoadBalancerBEAddressPool')]",

> +    "lbProbeID0": "[concat(variables('lbID0'),'/probes/FabricGatewayProbe')]",

> +    "lbHttpProbeID0": "[concat(variables('lbID0'),'/probes/FabricHttpGatewayProbe')]",

> +    "vmNodeType0Name": "[concat('NT1', variables('vmName'))]",

> +    "vmNodeType0Size": "Standard_A2"

> +  },

> +  "resources": [

> +    {

> +      "apiVersion": "2015-05-01-preview",

This should become a variable and also move to the GA API version - 2015-06-15

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/863/files#r46351555>.